week-2/Homework/mandatory/1-fix-functions.js

```diff
@@ -2,89 +2,86 @@
 // Look at the tests and see how you can fix them.
 
 function mood() {
-  let isHappy = true;
+    let isHappy = true;
 
-  if (isHappy) {
-    return "I am happy";
-  } else {
-    return "I am not happy";
-  }
+    if (!isHappy) {
+        return "I am happy";
+    } else {
+        return "I am not happy";
+    }
 }
 
 function greaterThan10() {
-  let num = 10;
-  let isBigEnough;
-
-  if (isBigEnough) {
-    return "num is greater than or equal to 10";
-  } else {
-    return "num is not big enough";
-  }
+    let num = 10;
+    let isBigEnough = 20
```

Comment:
I know this may not be much clear
but when you read a variable name starts with isSomething. it means usually it's a boolean variable
``` let isBigEnough=num>=10```